src/components/tx/confirmation-views/ChangeThreshold/index.tsx

```diff
@@ -5,10 +5,21 @@ import commonCss from '@/components/tx-flow/common/styles.module.css'
 import useSafeInfo from '@/hooks/useSafeInfo'
 import { ChangeThresholdReviewContext } from '@/components/tx-flow/flows/ChangeThreshold/context'
 import { ChangeSignerSetupWarning } from '@/features/multichain/components/SignerSetupWarning/ChangeSignerSetupWarning'
+import { SettingsInfoType, type TransactionDetails } from '@safe-global/safe-gateway-typescript-sdk'
+import { isSettingsChangeTxInfo } from '@/utils/transaction-guards'
 
-function ChangeThreshold() {
+interface ChangeThresholdProps {
+  txDetails?: TransactionDetails
+}
+
+function ChangeThreshold({ txDetails }: ChangeThresholdProps) {
   const { safe } = useSafeInfo()
   const { newThreshold } = useContext(ChangeThresholdReviewContext)
+  const threshold =
+    txDetails &&
+    isSettingsChangeTxInfo(txDetails.txInfo) &&
+    txDetails.txInfo.settingsInfo?.type === SettingsInfoType.CHANGE_THRESHOLD &&
+    txDetails.txInfo.settingsInfo.threshold
```

Comment:
Can we use the `isChangeThresholdView` guard instead?

Reply:
Sure, didn't know we have this one ehehe